net-misc/unison/unison-2.51.5.ebuild

```diff
@@ -17,7 +17,10 @@ IUSE="debug doc gtk +ocamlopt threads"
 BDEPEND="dev-lang/ocaml:=[ocamlopt?]
 	doc? ( app-text/dvipsk
 		app-text/ghostscript-gpl
-		dev-texlive/texlive-latex )"
+		dev-texlive/texlive-latex
+		dev-tex/hevea
+		www-client/lynx
```

Comment:
I haven't looked at unison's build system yet. But could you explain where and why lynx is needed?